src/content/en/fundamentals/accessibility/semantics-aria/index.md

```diff
@@ -0,0 +1,160 @@
+managed with native HTML. It works by allowing you to specify attributes that
+modify the way an element is translated into the accessibility tree. Let's look
+at an example.
+
+In the following snippet, we use a list item as a kind of custom checkbox. The
+CSS "checkbox" class gives the element the required visual characteristics.
+
+
+    <li tabindex="0" class="checkbox" checked>
+      Receive promotional offers
+    </li>
+    
+
+While this works fine for sighted users, a screen reader will give no indication
+that the element is meant to be a checkbox, so low-vision users may miss the
+element entirely.
+
+Using ARIA attributes, however, we can give the element the missing information
+so the screen reader can properly interpret it. Here, we've added the `role` and
+`aria-checked` attributes to explicitly identify the element as a checkbox and
+to specify that it is checked by default. The list item will now be added to the
+accessibility tree and a screen reader will correctly report it as a checkbox.
+
+
+    <li tabindex="0" class="checkbox" role="checkbox" checked aria-checked="true">
+      Receive promotional offers
+    </li>
+    
+
+>Later, we will cover the list of ARIA attributes and when to use them.
+
+ARIA works by changing and augmenting the standard DOM accessibility tree.
+
+![the standard DOM accessibility tree](imgs/acctree1.jpg)
+
+![the ARIA augmented accessibility tree](imgs/acctree2.jpg)
+
+Although ARIA allows us to subtly (or even radically) modify the accessibility
+tree for any element on the page, that is the only thing it changes. **ARIA
+doesn't augment any of the element's inherent behavior**; it won't make the
+element focusable or give it keyboard event listeners. That is still part of our
+development task.
+
+It's important to understand that there is no need to redefine default
+semantics. Regardless of its use, a standard HTML `<input type="checkbox">`
+element doesn't need an additional `role="checkbox"` ARIA attribute to be
+correctly announced.
+
+It's also worth noting that certain HTML elements have restrictions on what ARIA
+roles and attributes can be used on them. For example, a standard `<input
+type="text">` element may not have any additional role/attribute applied to it.
+
+>See the <a href="https://www.w3.org/TR/html-aria/#sec-strong-native-semantics"
+target="_blank">ARIA in HTML spec</a> for more information.
+
+Let's see what other capabilities ARIA has to offer.
+
+## What can ARIA do?
+
+As you saw with the checkbox example, ARIA can modify existing element semantics
+or add semantics to elements where no native semantics exist. It can also
+express semantic patterns that don't exist at all in HTML, like a menu or a tab
+panel. Often, ARIA lets us create widget-type elements that wouldn't be possible
+with plain HTML.
+
+ - For example, ARIA can add extra label and description text that is only
+   exposed to assistive technology APIs.
+
+```
```

Comment:
Similar to previous item. Can't get code blocks to work inside of lists unless using backticks